### ISSUE TITLE
Bug when building the content_hash for a new issue

### ISSUE THREAD

**Issue by troessner** [andyw8/codeclimate-engine-rb#6]
When building an issue the [content hash](https://github.com/andyw8/codeclimate-engine-rb/blob/master/lib/cc_engine/issue.rb#L47) looks like this:

``` Ruby
      {
        body: {
          content: content
        }
      }
```

Looking at the actual specs: 
- The [issue example](https://github.com/codeclimate/spec/blob/master/SPEC.md#issues) 
- The [content definition](https://github.com/codeclimate/spec/blob/master/SPEC.md#contents)

I believe the content_hash should be rather like this:

``` Ruby
      {
        content: {
          body: content
        }
      }
```

which also makes more sense if you think about it ;)

**Comment by andyw8** [comment]
Oops, yes, fixing...

**Comment by andyw8** [comment]
Fixed in 0.3.1

**Comment by troessner** [comment]
Very fast fix, thanks!
